src/Tizen.Nlp/Tizen.Nlp/NaturalLanguageProcess.cs

```diff
@@ -0,0 +1,279 @@
+                requestid = int.Parse((string)msg.GetItem("request_id"));
+                e.RequestId = requestid;
+                e.Message = result;
+                _mapsWordTokenize[requestid]?.Invoke(e);
+            }
+            else if (msg.GetItem("command").Equals("pos_tag"))
+            {
+                result.Add("token", (string[])msg.GetItem("return_token"));
+                result.Add("tag", (string[])msg.GetItem("return_tag"));
+                requestid = int.Parse((string)msg.GetItem("request_id"));
+                e.RequestId = requestid;
+                e.Message = result;
+                _mapsPosTag[requestid]?.Invoke(e);
+            }
+            else if (msg.GetItem("command").Equals("ne_chunk"))
+            {
+                result.Add("token", (string[])msg.GetItem("return_token"));
+                result.Add("tag", (string[])msg.GetItem("return_tag"));
+                requestid = int.Parse((string)msg.GetItem("request_id"));
+                e.RequestId = requestid;
+                e.Message = result;
+                _mapsNamedEntity[requestid]?.Invoke(e);
+            }
+            else if (msg.GetItem("command").Equals("lemmatize"))
+            {
+                result.Add("token", (string[])msg.GetItem("return_token"));
+                requestid = int.Parse((string)msg.GetItem("request_id"));
+                e.RequestId = requestid;
+                e.Message = result;
+                _mapsLemmatize[requestid]?.Invoke(e);
+            }
+            else if (msg.GetItem("command").Equals("langdetect"))
+            {
+                result.Add("token", (string[])msg.GetItem("return_token"));
+                requestid = int.Parse((string)msg.GetItem("request_id"));
+                e.RequestId = requestid;
+                e.Message = result;
+                _mapsLangDetect[requestid]?.Invoke(e);
+            }
+            else
+            {
+                return;
+            }
+            Log.Debug(_tag, "done");
+        }
+
+        /// <summary>
+        /// An construct method  to connect remote tidl service.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public NaturalLanguageProcess()
+        {
+            _tag = Application.Current.ApplicationInfo.ApplicationId;
+            Log.Debug(_tag, "msg construct started");
+            _msg = new Message(ServiceId);
+            Log.Debug(_tag, "msg construct success");
+            _noti.Received += ResultReceived;
+            Log.Debug(_tag, "notify callback be assigned");
+            _msg.Connected += (sender, e) =>
+            {
+                Log.Debug(_tag, "start to register");
+                _msg.CoRegister(Application.Current.ApplicationInfo.ApplicationId, _noti);
+                Log.Debug(_tag, "connected callback be called");
+            };
+            _msg.Rejected += (sender, e) =>
+            {
+                Log.Debug(_tag, "rejected callback be called");
```

Comment:
Please, handle rejected case. 
If connection was rejected, it should be notify to user(app developer)